solidity/test/XConversions.js

```diff
@@ -73,6 +73,7 @@ contract("XConversions", accounts => {
             const path = erc20TokenBntPath
             const amount = web3.toWei(1)
 
+            await erc20Token.approve(bancorNetwork.address, 0, { from: accounts[5] })
```

Comment:
@yudilevi : This line implies a stateful test. How come the allowance is larger than 0 (to require executing this "0-approve" transaction)? We should strive for stateless tests ("it"s).

Reply:
this is deliberately there in order to ensure we operate in a stateless state.
so even if there are other tests that might have done it, this line will reset the approval just in case it was forgotten in a different test.